### PR TITLE
Add configurability to default scrubs

### PR DESCRIPTION
**Pull request by etceterum**
We've found a need to disable some of the default scrubs in our project so we forked your repo and implemented it - and now would like to give back.

### CODE REVIEW

File: lib/logstop.rb [hunk 1]
```diff
@@ -14,19 +14,34 @@ module Logstop
...
+    password: true,
+    email: true,
+    credit_card: true,
+    phone: true,
+    ssn: true,
+    ip: false,
+    scrubber: nil,
+  }.freeze
+
+  def self.scrub(msg, **options)
```
Comment (ankane) [inline lib/logstop.rb:27]: Let's use actual keyword arguments instead of `**options`.

File: lib/logstop/formatter.rb [hunk 2]
```diff
@@ -2,14 +2,13 @@
 
 module Logstop
   class Formatter < ::Logger::Formatter
-    def initialize(formatter = nil, ip: false, scrubber: nil)
+    def initialize(formatter = nil, **options)
```
Comment (ankane) [inline lib/logstop/formatter.rb:5]: Let's keep keyword arguments here as well.

File: test/logstop_test.rb [hunk 3]
```diff
@@ -12,13 +12,26 @@ def test_email
     assert_filtered "test-test@example.org"
     assert_filtered "test@example.us"
     assert_filtered "test@example.science"
+    refute_filtered "test@example.org", email: false
```
Comment (ankane) [inline test/logstop_test.rb:15]: I think we can test this functionality with only a single assertion for each option.

File: README.md [hunk 4]
```diff
@@ -15,6 +15,15 @@ By default, scrubs:
 - Social Security numbers (SSNs)
 - passwords in URLs
 
+To disable any of the default scrubs above, use:
+```ruby
+Logstop.guard(logger, email: false)
+Logstop.guard(logger, phone: false)
+Logstop.guard(logger, credit_card: false)
+Logstop.guard(logger, ssn: false)
+Logstop.guard(logger, password: false)
```
Comment (ankane) [inline README.md:24]: Let's go with `url_password` for this one.

**Review by ankane — COMMENTED** [review]
Hey @etceterum, thanks for the PR! Overall, looks great. Left a few comments inline.

**Comment by etceterum** [comment]
@ankane Thank you for the feedback and sorry for being slow with my response. I will address your comments in the next few days

**Comment by etceterum** [comment]
I believe I have all of your findings addressed now @ankane, please let me know if any other changes are needed.

**Comment by ankane** [comment]
Looks great, thanks @etceterum!

**Comment by etceterum** [comment]
Thanks very much @ankane! I don't see the new version at rubygems (eager to switch us back from fork to the official branch 😄 - and I am now wondering if that's because I forgot to bump the version inside `version.rb` 🤔

**Comment by ankane** [comment]
Just pushed a new release.

**Pull request merged**